constraints.txt

```diff
@@ -102,3 +102,4 @@ urllib3==1.18.1
 virtualenv==15.1.0
 wheel==0.30.0
 wrapt==1.10.10
+git+https://github.com/rcbops/rpc_component@776c7d7ddc96fd62b503c78c980256906db5a6fd#egg=rpc_component
```

Comment:
Would it be cleaner to have a separate constraints file for rpc_component itself? We never install rpc_component into the standard gating venv so it feels a bit odd having it in here. Also, the constraints we have here don't match what was in https://github.com/mattt416/rpc-metadata/blob/master/constraints.txt -- ie. differing GitPython version and schema is unconstrained.

Reply:
While it's not currently required in the standard gating venv, I wouldn't be surprised if that changes. I thought it would be easier to keep everything in step so there aren't any issues if that happens. I've updated the patch to add a constraint for schema and updated GitPython (it's only used be release related code) but I'm I don't have an issue with creating a separate constraints file if you prefer.

Reply:
As mentioned in chat, you can't install requirements.txt into a python3 venv so it doesn't make much sense bundling these together.